src/Models/Properties/ElementProperties/BaseProperty.cs

```diff
@@ -76,5 +76,11 @@ public partial class BaseProperty
         public string SummaryLabel { get; set; } = string.Empty;
 
         public string Warning { get; set; } = string.Empty;
+
+        public string OptionalIf { get; set; } = string.Empty;
```

Comment:
Is this the questionId of the element being compared to? If so, could we change the name to OptionalIfQuestionId instead to make it clearer?